components/stub-generation/block.tsx

```diff
@@ -12,24 +18,83 @@ const Item = (props: { title: string, description: string, SvgComponent: Functio
 export default function ProblemBlocks() {
   return (
     <section className="relative">
-      <div className="absolute inset-0 top-1/2 md:mt-24 lg:mt-0 bg-gradient-to-b from-secondary-500 to-secondary-100  pointer-events-none" aria-hidden="true" />
+      <div
+        className="absolute inset-0 top-1/2 md:mt-24 lg:mt-0 bg-gradient-to-b from-secondary-500 to-secondary-100  pointer-events-none"
+        aria-hidden="true"
+      />
       <div className="absolute left-0 right-0 bottom-0 m-auto w-px p-px h-20 bg-gray-200 transform translate-y-1/2" />
-
       <div className="relative max-w-6xl mx-auto px-4 sm:px-6">
-        <div className="pb-12 md:pb-20">
-          <div className=" w-full text-center pb-6 md:pb-10">
-            <h1 className="h1 mb-4 text-secondary-300 whitespace-nowrap">Smart generation - Enhanced development</h1>
+        <div className="py-12 md:py-20">
+          <div className="w-full text-center pb-6 md:pb-10">
+            <h1 className="h1 mb-4 text-secondary-300">
+              Smart generation - Enhanced development
+            </h1>
             <div className="flex justify-center w-full">
               <h3 className="h3 text-xl text-gray-600 flex justify-center items-center">
-
                 &nbsp; 👋 Say Hi to 👋 &nbsp;
-    
               </h3>
             </div>
+            <div className="flex text-gray-600 pt-3 pl-6 pr-6">
+              <svg
```

Comment:
Please create separate img for svg and use it that would be more cleaner and Also these arrow placement is not correctly aligned with the text please check
![Screenshot from 2024-04-03 18-11-41](https://github.com/keploy/website/assets/78696767/e98368e6-a8c2-4747-b78f-8532db4ae8fd)

Reply:
Hey @Hermione2408 just to confirm are you asking to replace the arrow svgs with their respective images?

Reply:
> Hey @Hermione2408 just to confirm are you asking to replace the arrow svgs with their respective images?

you can create a separate file for this svg then use that here as image

Reply:
okay!